src/main/java/core/basesyntax/service/impl/CsvReader.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReader implements Reader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
```

Comment:
The exception message 'Can't read file' is not informative enough. It's better to include the file path in the message to provide more context about the error. For example: throw new RuntimeException("Can't read file at path: " + filePath, e);

Reply:
corrected